OperatingSystems.md

```diff
@@ -158,6 +158,23 @@ Type|Macro
 ---|---
 Identification|`__ECOS`
 
+## [Emscripten](https://en.wikipedia.org/wiki/Emscripten) ##
+
+Type|Macro|Format|Description|Availability|Header
+---|---|---|---|---|---
+Identification|`EMSCRIPTEN`| | |Defined as of 1.0.1a, gone as of 1.30.6|None
+Identification|`__EMSCRIPTEN__`| | |Defined as of 1.3.7|None
+Version|`__EMSCRIPTEN_major__`|V|V = Major version|Defined as of 1.21.4|`emscripten/version.h`
```

Comment:
Header name goes to Description column in the format of:
```V = Major version<br><br>Must be included from <header.h>```

Availability goes to Description column as well. It would be nice if you will include links to Emscripten documentation.

Reply:
Done!